pkg/auth/tokens/token_util_test.go

```diff
@@ -213,3 +213,59 @@ func expireToken(token *v3.Token) *v3.Token {
 	newToken.TTLMillis = 1
 	return newToken
 }
+
+func TestIsIdleExpired(t *testing.T) {
```

Comment:
We can this test in parallel.
```suggestion
func TestIsIdleExpired(t *testing.T) {
    t.Parallel()
```